docs/concepts/extension_points.rst

```diff
@@ -105,45 +103,66 @@ Here are the different integration points that python plugins can use:
    * - Django App Plugin (``lms.djangoapp`` and ``cms.djangoapp``)
-     - Adopt, Stable
+     - ⚠️ **Provisional**
      - A "Django app plugin" is a self-contained Django `Application`_ that can define models (MySQL tables), new REST APIs, signal listeners, asynchronous tasks, and more. Even some parts of the core platform are implemented as Django app plugins, for better separation of concerns (``announcements``, ``credentials``, ``grades``, etc.) Read the `Django app plugin documentation`_ to learn more.
 
        Plugins can also inject custom data into django template contexts, to affect standard pages delivered by the core platform. See `Plugin Contexts`_ to learn more.
    * - Course tab (``openedx.course_tab``)
-     - Hold, Stable
+     - 🕰️ **Legacy**
      - A course tab plugin adds a new tab shown to learners within a course. ``courseware``, ``course_info``, and ``discussion`` are examples of built-in tab plugins. Read the `course tabs documentation`_ to learn more.
 
        This API may be changing soon with the new Courseware microfrontend implementation.
    * - Course tool (``openedx.course_tool``)
-     - Hold, Stable
+     - 🕰️ **Legacy**
      - The course home page (the landing page for the course) includes a "Course Tools" section that provides links to "tools" associated with the course. Examples of course tool plugins included in the core are reviews, updates, and bookmarks. See |course_tools.py|_ to learn more.
 
        This API may be changing soon with the new Courseware microfrontend implementation.
    * - Custom registration form app (``REGISTRATION_EXTENSION_FORM`` Django setting in the LMS)
-     - Trial, Stable
+     - ⚠️ **Provisional**
      - By default, the registration page for each instance of Open edX has fields that ask for information such as a user’s name, country, and highest level of education completed. You can add custom fields to the registration page for your own Open edX instance. These fields can be different types, including text entry fields and drop-down lists. See `Adding Custom Fields to the Registration Page`_.
    * - Learning Context (``openedx.learning_context``)
-     - Trial, Limited
+     - ⚠️ **Provisional**
      - A "Learning Context" is a course, a library, a program, a blog, an external site, or some other collection of content where learning happens. If you are trying to build a totally new learning experience that's not a type of course, you may need to implement a new learning context. Learning contexts are a new abstraction and are only supported in the nascent Learning-Core-based XBlock runtime. Since existing courses use modulestore instead of Learning Core, they are not yet implemented as learning contexts. However, Learning-Core-based content libraries are. See |learning_context.py|_ to learn more.
    * - User partition scheme (``openedx.user_partition_scheme`` and ``openedx.dynamic_partition_generator``)
-     - Unknown, Stable
+     - ⚠️ **Provisional**
```

Comment:
I would have thought this was stable, based on 2U usage. But, I'm also aware of significant bugs in this.

Reply:
Right, this one has extensive usage, but the API itself is poorly defined and has bugs. We're not going to remove the API (in fact, we'll probably improve it) but I didn't want to encourage folks to jump into using it like the other Stable APIs.